handlers.go

```diff
@@ -73,6 +73,11 @@ func pingHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("pong\n"))
 }
 
+// healthHandler is a simple handler to test if daemon is UP.
+func healthHandler(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusOK)
```

Comment:
@Issif Do you want me to respond a JSON as you suggested? I think the `200` status code is enough, it will tell the same thing as `{"status": "ok"}`.  Moreover, many health check features (Kubernetes HTTPGet Probe, Google Load Balancer, etc) checked the status code and won't check the resp body.

WDYT?

Reply:
It is good to send the body as well, because in the future we can add more info, for example, we can check if the AWS is still valid and respond on that as well like

```json
{
  "status": "ok",
  "aws": "ok",
  "pagerduty": "ok"
  ...
}
```

for now the `/healtz` is a dummy which is fine

Reply:
We also can add the `ready` endpoint to say that the APP is ready, but for that, we will need a bit more logic

Reply:
Thanks for your reply. 

> It is good to send the body as well, because in the future we can add more info, for example, we can check if the AWS is still valid and respond on that as well like

Totally agree. 

> for now the /healtz is a dummy which is fine

Okay, let's go on for now 👍 
Thank you.